firefox-ios/Client/Frontend/TrackingProtection/TrackingProtectionButton.swift

```diff
@@ -0,0 +1,108 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Common
+import ComponentLibrary
+
+public struct TrackingProtectionButtonViewModel {
+    let title: String
+    let a11yIdentifier: String
+
+    init(title: String = "",
+         a11yIdentifier: String = "") {
+        self.title = title
+        self.a11yIdentifier = a11yIdentifier
+    }
+}
+
+class TrackingProtectionButton: ResizableButton, ThemeApplicable {
```

Comment:
@petruSt24 do you have a screenshot of this UI view? Is this something specific to enhanced protection or do we foresee it being used generically and should live in the component library?

Reply:
<img width="340" alt="image" src="https://github.com/mozilla-mobile/firefox-ios/assets/126153378/b7f3ba85-bf08-46ef-83c2-c40b33055475">
@cyndichin I was thinking if it should go into the component library but I haven't seen it being used anywhere else even in the enhanced tracking protection flow, I'd add it at the usecase no.2, if it ever arises

Reply:
okay thank you! we can leave it as is. Seems like its generic enough where we can move it in the library if we wanted to. Thanks!